tests/dataset/test_study_index.py

```diff
@@ -0,0 +1,69 @@
+"""Test study index dataset."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import dbldatagen as dg
+import pytest
+
+from otg.common.schemas import parse_spark_schema
+from otg.dataset.study_index import StudyIndex
+
+if TYPE_CHECKING:
+    from pyspark.sql import SparkSession
+
+
+@pytest.fixture
+def mock_study_index(spark: SparkSession) -> StudyIndex:
+    """Mock v2g dataset."""
+    schema = parse_spark_schema("studies.json")
+
+    data_spec = (
+        dg.DataGenerator(
+            spark,
+            rows=400,
+            partitions=4,
+            randomSeedMethod="hash_fieldname",
+            name="v2g",
```

Comment:
Why is the name `v2g`? I guess this is a typo, but in general, what is the importance of this field? Taking a quick look at the [documentation](https://databrickslabs.github.io/dbldatagen/public_docs/APIDOCS.html), this field is optional and doesn't do much.

Reply:
Defaults to None. I'll get rid off all of them